pyproject.toml

```diff
@@ -198,14 +198,13 @@ omit = [
 [tool.pytest.ini_options]
 doctest_optionflags = 'NUMBER ELLIPSIS'
 filterwarnings = [
-  'error::pyvista.PyVistaDeprecationWarning',
-  'ignore:.*Given trait value dtype "float64":UserWarning', # bogus numpy ABI warning (see numpy/#432)
-  'ignore:.*The NumPy module was reloaded*:UserWarning',    # bogus numpy ABI warning (see numpy/#432)
-  'ignore:.*numpy.dtype size changed.*:RuntimeWarning',     # bogus numpy ABI warning (see numpy/#432)
-  'ignore:.*numpy.ufunc size changed.*:RuntimeWarning',     # bogus numpy ABI warning (see numpy/#432)
-  'ignore::DeprecationWarning',
-  'ignore::FutureWarning',
-  'ignore::PendingDeprecationWarning',
+  'error',
+  'ignore:.*Given trait value dtype "float64":UserWarning',                                   # bogus numpy ABI warning (see numpy/#432)
+  'ignore:.*The NumPy module was reloaded*:UserWarning',                                      # bogus numpy ABI warning (see numpy/#432)
+  'ignore:.*numpy.dtype size changed.*:RuntimeWarning',                                       # bogus numpy ABI warning (see numpy/#432)
+  'ignore:.*numpy.ufunc size changed.*:RuntimeWarning',                                       # bogus numpy ABI warning (see numpy/#432)
+  'ignore:Points is not a float type.*:UserWarning',
+  'ignore:pyvista test generated image dir.* does not yet exist.  Creating dir.:UserWarning',
```

Comment:
Harmless ignore since the warning is only to inform the user a folder is being created by pytest-pyvista